zulipterminal/ui_tools/views.py

```diff
@@ -1678,6 +1682,17 @@ def keypress(self, size: urwid_Size, key: str) -> str:
                 time_mentions=self.time_mentions,
             )
             return key
+        elif is_command_key("COPY_MESSAGE", key):
+            rendered_content, *_ = MessageBox.transform_content(
+                self.msg["content"], self.controller.model.server_url
+            )
+            content = []
+            for word in rendered_content[1]:
+                if isinstance(word, tuple):  # if msg content has markup
+                    content.append(word[1])
+                else:
+                    content.append(word)
+            self.controller.copy_to_clipboard("".join(content), "Message Content")
```

Comment:
The principle here is just fine, but we do perform a few extra adjustments to content which may not follow the principles that you assume here.

That's likely what gave rise to the crash I reported in the stream.